doc/user/content/sql/begin.md

```diff
@@ -1,55 +1,166 @@
 ---
 title: "BEGIN"
+description: "`BEGIN` starts a transaction block."
 menu:
   main:
     parent: "commands"
 ---
 
-`BEGIN` starts a transaction block.
+{{% txns/txn-details %}}
```

Comment:
This is below: https://github.com/MaterializeInc/materialize/pull/32321/files#diff-5ab908fd5d660e3e17d3d1822788fadf1bc3ecbe3a015507c0d69054b69e2d51